src/solver/sLR.ml

```diff
@@ -527,7 +527,7 @@ let _ =
   Selector.add_solver ("widen2",  (module PostSolver.EqIncrSolverFromEqSolver (W2)));
   Selector.add_solver ("widen3",  (module PostSolver.EqIncrSolverFromEqSolver (W3)));
   let module S2 = TwoPhased (struct let ver = 1 end) in
-  Selector.add_solver ("two",  (module PostSolver.EqIncrSolverFromEqSolver (S2)));
+  Selector.add_solver ("two",  (module PostSolver.EqIncrSolverFromEqSolver (S2))); (* TODO: broken even on 00-sanity/01-assert *)
```

Comment:
Broken how? if it doesn't work at all, we should remove it --- no point in keeping broken stuff around!

Reply:
Broken as it computes bottom everywhere, like `topdown_term`. The latter I managed to easily fix here, but I don't know the SLR solvers well enough to tell if there's a similar silly issue somewhere.
It's the only SLR variation with such issue (and most share code), so there's not much to remove code-wise. I commented this `add_solver` out though, so Goblint cannot be executed with this solver.